build.gradle.kts

```diff
@@ -1,7 +1,7 @@
 plugins {
     id("com.android.application") version "7.3.1" apply false
     id("com.android.library") version "7.3.1" apply false
-    id("org.jetbrains.kotlin.android") version "1.7.20" apply false
+    id("org.jetbrains.kotlin.android") version "1.7.21" apply false
```

Comment:
Please unbump the version number. The Dependabot PR in your repo (`#3`) shows that the checks fail (which also applies in this PR) so you shouldn't merge them if they do.

Reply:
I know I should've just ignored these PRs until it got updated, oh well ^^;

Anyways, yeah applying. where do I locate the file?